.github/workflows/build_all.yml

```diff
@@ -33,6 +33,7 @@ env:
 jobs:
   build_windows_cpp:
     runs-on: windows-latest
+    permissions: read-all
```

Comment:
Put the read-all once at the top of the file
https://github.com/oneapi-src/distributed-ranges/blob/d4cc293840f748fdc1a208b82598254c21f721a3/.github/workflows/pr.yml#L6